Lib/fontTools/fontBuilder.py

```diff
@@ -0,0 +1,548 @@
+from __future__ import print_function, division, absolute_import
+from __future__ import unicode_literals
+
+__all__ = ["FontBuilder"]
+
+"""
+The `FontBuilder` class is a convenient helper to construct working TTF or OTF
+fonts from scratch.
+
+Here is how to build a minimal TTF:
+
+```python
+from fontTools.fontBuilder import FontBuilder
+from fontTools.pens.ttGlyphPen import TTGlyphPen
+
+def drawTestGlyph(pen):
+    pen.moveTo((100, 100))
+    pen.lineTo((100, 1000))
+    pen.qCurveTo((200, 900), (400, 900), (500, 1000))
+    pen.lineTo((500, 100))
+    pen.closePath()
+
+fb = FontBuilder(1024, isTTF=True)
+fb.setupGlyphOrder([".notdef", ".null", "A", "a"])
+fb.setupCharacterMap({65: "A", 97: "a"})
+
+advanceWidths = {".notdef": 600, "A": 600, "a": 600, ".null": 600}
+
+familyName = "HelloTestFont"
+styleName = "TotallyNormal"
+nameStrings = dict(familyName=dict(en="HelloTestFont", nl="HalloTestFont"),
+                   styleName=dict(en="TotallyNormal", nl="TotaalNormaal"))
+nameStrings['psName'] = familyName + "-" + styleName
+
+pen = TTGlyphPen(None)
+drawTestGlyph(pen)
+glyph = pen.glyph()
+glyphs = {".notdef": glyph, "A": glyph, "a": glyph, ".null": glyph}
+fb.setupGlyf(glyphs)
+fb.calcGlyphBounds()
```

Comment:
this call to `fb.calcGlyphBounds()` is not needed here, no? The `setupGlyf` method does it already by default.